### DISCUSSION TITLE
Theme colors don't seem to be rendered correctly.

### DISCUSSION THREAD

**Discussion by Ackerven** [vuepress-theme-hope/vuepress-theme-hope#4550] [Q&A] [answered]
![5afbc6becc5b522c3d553e4d4159eb1](https://github.com/user-attachments/assets/7db21a03-fbfa-4bd4-86ae-8a6bfc9c226c)
![image](https://github.com/user-attachments/assets/1adde21a-1bc5-4198-95bd-4379250f5a61)

I got the color code from here: [https://www.zkcoi.com/idea/design/2228.html](https://www.zkcoi.com/idea/design/2228.html)

**Comment by Mister-Hope** [comment, marked as answer]
This is expected to fit the a11y requirements.

If you want to manually override the generated colors, set them implicitly in palette
